build/ProjectReunion-Create-VSIX.yml

```diff
@@ -0,0 +1,96 @@
+parameters:
+  - name: "ReunionVersion"
+    type: string
+    default: ''
+  - name: "ReunionFoundationVersion"
+    type: string
+    default: ''
+  - name: "ReunionDWriteVersion"
+    type: string
+    default: ''
+  - name: "ReunionWinUIVersion"
+    type: string
+    default: ''
+  - name: "releaseBuild"
+    type: boolean
+    default: True
+
+jobs:
+
+- job: CreatingVsix
+  pool:
+    vmImage: 'windows-2019'
+
+  variables:
+    ${{if eq(parameters.releaseBuild, True) }}:
+      VSIXBuildArgs: '/p:"RestoreSources=https://api.nuget.org/v3/index.json;https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.Internal.ReleaseSigned/nuget/v3/index.json"'
+    ${{if eq(parameters.releaseBuild, False) }}:
+      VSIXBuildArgs: ''
+      
+  steps:
+  - task: NuGetAuthenticate@0
+    inputs:
+      nuGetServiceConnections: Internal-ReleaseSigned
+
+  # NuGetCommand@2 
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    displayName: 'NuGet restore'
```

Comment:
Maybe for the next time you could consolidate the two different restores steps that are happening on this step and the next step.